vispy/visuals/graphs/layouts/__init__.py

```diff
@@ -0,0 +1,52 @@
+import inspect
+
+from .random import random
+from .circular import circular
+from .force_directed import fruchterman_reingold
+
+
+_layout_map = {
+    'random': random,
+    'circular': circular,
+    'force_directed': fruchterman_reingold,
+    'spring_layout': fruchterman_reingold
+}
+
+AVAILABLE_LAYOUTS = _layout_map.keys()
+
+
+def get(name, *args, **kwargs):
+    """
+    Retrieve a graph layout
+
+    Some graph layouts accept extra options. Please refer to their
+    documentation for more information.
+
+    Parameters
+    ----------
+    name : string
+        The name of the layout. The variable :data`AVAILBLE_LAYOUTS`
```

Comment:
Not sure if it's necessary either since it's not actually documented, but it should be `:data:` in any case. And `AVAILABLE` is typo'd here.